tracking/translations_parser/utils.py

```diff
@@ -50,14 +50,16 @@
 def parse_tag(tag, sep="_"):
     # First try to parse a simple training label
     match = TRAIN_LABEL_REGEX.match(tag)
-    if match is not None:
-        return match.groupdict()["model"], None, None, None
-    # Else try to parse an evaluation label with importer, dataset and auugmentation
-    match = EVAL_REGEX.match(tag)
+    if match is None:
+        # Else try to parse an evaluation label with importer, dataset and auugmentation
+        match = EVAL_REGEX.match(tag)
     if not match:
         raise ValueError(tag)
     groups = match.groupdict()
-    return groups["model"], groups["importer"], groups["dataset"], groups["aug"]
+    # Always keep the index in the model name (teacher-1/2 → teacher-1)
```

Comment:
FYI we're going to rename teacher 1/2 to teacher 1 not to invalidate the caches when we want to train the second part of the teacher ensemble after an experiment with only one teacher. See #544

Reply:
So maybe it's worth adding support for the names without `/2` right away. @gregtatum We'll need to sync the work on #544 with this.

Reply:
Hopefully this is already handled by the parser from #449 (covered [here](https://github.com/mozilla/firefox-translations-training/pull/589/files#diff-cc6b34cc2db0bc83dd40f5ec44f20ef4d2d81b9d01c8f919a845b00b78151b2bL22)).

|full name | corresponding run |
|-|-|
| `teacher_flores_devtest` | `teacher-1` |
| `teacher-1_flores_devtest`| `teacher-1`|
| `teacher-2_flores_devtest`| `teacher-1`|
| `teacher-flores_devtest_1/2`| `teacher-1`|
| `teacher-flores_devtest_2/2`| `teacher-2`|